### DISCUSSION TITLE
Prioritize certain maschines depending on the employee doing a task?

### DISCUSSION THREAD

**Discussion by MathiasMantai** [frePPLe/frepple#630] [Q&A]
Hi,

I'm trying to realize the following: Every maschine can be operated by multiple employees, but certain employees are prioritized to work with certain maschines. Is there a way to make sure that the prioritized employee is chosen for the task if they are free?

Thank you for the help.

**Comment by jdetaeye** [comment]
The skill model is what you need:
https://frepple.com/docs/current/user-guide/examples/resource/resource-skills.php
https://frepple.com/docs/current/user-guide/model-reference/skills.php
https://frepple.com/docs/current/user-guide/model-reference/resource-skills.php

The resource skill table defines what which resources/employees have a particular skill.    The table has a priority field that allows you to set the preferred resources when a job with a certain skill needs to be planned.
